src/Sarif.Viewer.VisualStudio.UnitTests/OutputWindowTracerListenerTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Text;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.VisualStudio.Shell.Interop;
+
+using Moq;
+
+using Xunit;
+
+namespace Microsoft.Sarif.Viewer.VisualStudio.UnitTests
+{
+    public class OutputWindowTracerListenerTests : SarifViewerPackageUnitTests
+    {
+        [Fact]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "VSTHRD010:Invoke single-threaded types on Main thread", Justification = "For unit tests")]
+        public void OutputWindowTracerListener_WriteTests()
+        {
+            // arrange
+            string currentOutputString = null;
+
+            var mockPane = new Mock<IVsOutputWindowPane>();
+            IVsOutputWindowPane pane = mockPane.Object;
+            mockPane
+                .Setup(p => p.OutputStringThreadSafe(It.IsAny<string>()))
+                .Callback((string outputString) => currentOutputString = outputString);
+
+            var mockOutputWindow = new Mock<IVsOutputWindow>();
+            mockOutputWindow.Setup(o => o.CreatePane(ref It.Ref<Guid>.IsAny, It.IsAny<string>(), 1, 1)).Returns(0);
+            mockOutputWindow.Setup(o => o.GetPane(ref It.Ref<Guid>.IsAny, out pane));
+
+            // act
+            var outputWindowTraceListener = new OutputWindowTracerListener(mockOutputWindow.Object, "TestPane");
+
+            Trace.Write("Test log");
+
+            // assert
+            mockPane.Verify(p => p.OutputStringThreadSafe(It.IsAny<string>()), Times.Once);
+            currentOutputString.Should().Be("Test log");
+
+            Trace.WriteLine("More test logs");
+
+            mockPane.Verify(p => p.OutputStringThreadSafe(It.IsAny<string>()), Times.Exactly(2));
+            currentOutputString.Should().Be(Environment.NewLine + "More test logs");
```

Comment:
This doesn't follow the arrange/act/assert pattern. Something I have seen when you need to interleave act and assert is to instead combine the two with the comment:
`// act..assert`

This way, you clearly split up your setup process but don't add any confusion with the remainder of the test not matching the pattern.